scraper/src/youtube2zim/schemas.py

```diff
@@ -0,0 +1,94 @@
+from humps import camelize
+from pydantic import BaseModel
+
+
+class CamelModel(BaseModel):
+    """Model to transform Python snake_case into JSON camelCase."""
+
+    class Config:
+        alias_generator = camelize
+        populate_by_name = True
+
+
+class Author(CamelModel):
+    channel_id: str
+    channel_title: str
+    profile_path: str | None = None
+    banner_path: str | None = None
+
+
+class Subtitle(CamelModel):
+    """Class to serialize data about a YouTube video subtitle."""
+
+    code: str
+    name: str
+
+
+class Video(CamelModel):
+    """Class to serialize data about a YouTube video."""
+
+    id: str
+    title: str
+    description: str
+    author: Author
+    publication_date: str
+    video_path: str
+    thumbnail_path: str | None = None
+    subtitle_path: str | None = None
+    subtitle_list: list[Subtitle]
+    duration: str
+
+
+class VideoPreview(CamelModel):
+    """Class to serialize data about a YouTube video for preview."""
+
+    slug: str
+    id: str
+    title: str
+    thumbnail_path: str | None = None
+    duration: str
+
+
+class Playlist(CamelModel):
+    """Class to serialize data about a YouTube playlist."""
+
+    id: str
+    author: Author
+    title: str
+    description: str
+    publication_date: str
+    thumbnail_path: str | None = None
+    videos: list[VideoPreview]
+    count: int
```

Comment:
Rename it to `videos_count` to remove any ambiguities

Reply:
Renamed in [1193985](https://github.com/openzim/youtube/pull/219/commits/119398503ebcc30b18d5accb33b46033d957ae90)